spec/spec.md

```diff
@@ -0,0 +1,174 @@
+# Verona Language Spec
+
+## Lexical Elements
+
+### Comments
+
+### Identifiers and Symbols
+
+- `_` as "don't care".
+
+### Strings
+
+### Constants
+
+- Boolean
+- Various integers and floats
+
+> How to talk about integer literals not having a type yet?
+
+### Lambdas
+
+### Object Literals
+
+- Open issues: code reuse, possible conflict with "real" `new`.
```

Comment:
?

Reply:
Currently the parser uses `new` for string literals, but that might not be the final choice, and we should decide.